### PR TITLE
feat: The HTTP module supports set Cookie

### PR DESCRIPTION
**Pull request by kkumar-gcc**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by devhaozi** [comment]
@kkumar-gcc tests failed

File: go.mod
Comment (hwbrzzl) [inline go.mod:1]: S: Please update go.sum

**Review by hwbrzzl — CHANGES_REQUESTED** [review]
The logic is good, please optimize the unit test. 👍

File: context_response_test.go [hunk 1]
```diff
@@ -299,6 +301,57 @@ func TestResponse(t *testing.T) {
...
 			expectBody: "Goravel",
 		},
+		{
+			name:       "WithoutCookie",
+			method:     "GET",
+			url:        "/without/cookie",
+			cookieName: "goravel",
+			setup: func(method, url string) error {
+				fiber.Get("/without/cookie", func(ctx contractshttp.Context) contractshttp.Response {
+					return ctx.Response().WithoutCookie("goravel").String(http.StatusOK, "Goravel")
```
Comment (hwbrzzl) [inline context_response_test.go:311]: S: Need to add cookie when making the request, the `WithoutCookie` method will be used.

Comment (kkumar-gcc) [reply context_response_test.go:311]: I am adding cookies here 
image

File: context_request_test.go [hunk 2]
```diff
@@ -1461,6 +1463,44 @@ func TestRequest(t *testing.T) {
...
 			expectBody: "Validate error: error",
 		},
+		{
+			name:       "Cookie",
+			method:     "GET",
+			url:        "/cookie",
+			cookieName: "user",
+			setup: func(method, url string) error {
+				fiber.Get("/cookie", func(ctx contractshttp.Context) contractshttp.Response {
+					return ctx.Response().Cookie(contractshttp.Cookie{
```
[the remark this answers is not in the document]
Comment (kkumar-gcc) [reply context_request_test.go]: Okay

File: context_response_test.go [hunk 3]
```diff
@@ -335,6 +388,15 @@ func TestResponse(t *testing.T) {
 				assert.Equal(t, test.expectHeader, strings.Join(resp.Header.Values("Hello"), ""))
 			}
 
+			if test.cookieName != "" {
+				cookies := resp.Cookies()
+				for _, cookie := range cookies {
+					if cookie.Name == test.cookieName {
+						assert.Equal(t, test.expectedCookieValue, cookie.Value)
+					}
+				}
```
Comment (hwbrzzl) [inline context_response_test.go:397]:
```suggestion
				exist := false
				for _, cookie := range cookies {
					if cookie.Name == test.cookieName {
					        exist := ture
						assert.Equal(t, test.expectedCookieValue, cookie.Value)
					}
				}
				assert.True(t, exist)
```

**Review by hwbrzzl — APPROVED** [review]
Great job 👍

**Pull request merged**